### ISSUE TITLE
quick links in Plots tab

### ISSUE THREAD

**Issue by jjgao** [cBioPortal/cbioportal#6651]
- IF copy-number & mRNA → expression vs copy-number box plot [mRNA vs CNA]
- IF methylation & mRNA → expression vs methylation scatter plot [mRNA vs methyl.]
- IF available → Mutation count vs FGA [Mut# vs FGA]
- IF cancer type detailed >1 and <=15 → Mutation count vs cancer type detailed [Mut# vs Dx]
- ELSIF cancer type detailed >15 → Mutation count vs cancer type [Mut# vs Dx]
- IF mRNA & cancer type detailed >1 → mRNA vs cancer type detailed [mRNA vs Dx]
- IF mRNA and mutation count > 0 → mRNA vs mutation type [mRNA vs mut type]
- IF mRNA & protein → mRNA vs protein scatter plot [Protein vs mRNA]
- IF treatments → sample order vs treatment IC50 [Tx Waterfall]

**Comment by jjgao** [comment]
Notifying @tmazor @schultzn @cBioPortal/product

**Comment by Sjoerd-van-Hagen** [comment]
I like the idea of having examples. My main concern is how we will implement this in a way that does not clutter up the interface, which already has a considerable number of bells and whistles.

**Comment by schultzn** [comment]
Here is a suggestion

![image](https://user-images.githubusercontent.com/11892704/66062870-38ff3e80-e510-11e9-9755-f45a752b04be.png)

**Comment by alisman** [comment]
@schultzn i think that works.  i do think they need a label ("Quick links?").  We have Quick Filters in study page.  Perhaps we can reuse the white bar under the nav.   @Luke-Sikina this can be your ticket.  i will work on making a more descriptive spec.

**Comment by alisman** [comment]
exactly.   i don't think it essential, but nice to follow convention.  they're also centered on study page.  could go either way on this.  and we don't have to decide now to start implementation.

**Comment by adamabeshouse** [comment]
@jjgao I think putting it in the URL would be definitely doable and wouldn't go stale so easily - we can use the data type enum, profile id, and gene id? the issue I think would be the URL would get really long..

**Comment by jjgao** [comment]
@adamabeshouse @alisman we should look into options for storing states in url and in session service. Maybe it's time to work it out? https://github.com/cBioPortal/cbioportal/issues/5525

Maybe for this ticket here, we don't have to worry about it as long as the users can click and get to the plots? Or is it a good opportunity to make saving state happen?

**Comment by adamabeshouse** [comment]
my sense is lets just focus on the feature, I think. saving state is more in depth and I think will take a while to shake out.

**Comment by jjgao** [comment]
@adamabeshouse that sounds good to me.

**Comment by Luke-Sikina** [comment]
Just starting this issue. Here's my summary after reading through this.
- create all valid quick links, as described in the main issue
- style the quick links similar to the study page, but also refer to the mock
- don't bother with the url for now, feature has value without it and there seems to be ongoing discussion about if / how

Please let me know if I missed anything.

**Comment by Sjoerd-van-Hagen** [comment]
> So something like this?
> 

Looks good but I would change 'Quick links' to 'Examples' or 'Example plots'.

**Comment by alisman** [comment]
👍

**Comment by tmazor** [comment]
@Luke-Sikina can we try this order?

IF cancer type detailed >1 and <=15 → Mutation count vs cancer type detailed [Mut# vs Dx]
ELSIF cancer type detailed >15 → Mutation count vs cancer type [Mut# vs Dx]
IF available → Mutation count vs FGA [Mut# vs FGA]
IF mRNA & cancer type detailed >1 → mRNA vs cancer type detailed [mRNA vs Dx]
IF mRNA and mutation count > 0 → mRNA vs mutation type [mRNA vs mut type]
IF copy-number & mRNA → expression vs copy-number box plot [mRNA vs CNA]
IF methylation & mRNA → expression vs methylation scatter plot [mRNA vs methyl]
IF mRNA & protein → mRNA vs protein scatter plot [Protein vs mRNA]
IF treatments → sample order vs treatment IC50 [Tx Waterfall]

**Comment by schultzn** [comment]
And maybe add FGA vs Cancer Type, so:

IF cancer type detailed >1 and <=15 → Mutation count vs cancer type detailed [Mut# vs Dx]
ELSIF cancer type detailed >15 → Mutation count vs cancer type [Mut# vs Dx]
IF cancer type detailed >1 and <=15 → FGA vs cancer type detailed [FGA vs Dx]
ELSIF cancer type detailed >15 → FGA vs cancer type [FGA vs Dx]
IF available → Mutation count vs FGA [Mut# vs FGA]
IF mRNA & cancer type detailed >1 → mRNA vs cancer type detailed [mRNA vs Dx]
IF mRNA and mutation count > 0 → mRNA vs mutation type [mRNA vs mut type]
IF copy-number & mRNA → expression vs copy-number box plot [mRNA vs CNA]
IF methylation & mRNA → expression vs methylation scatter plot [mRNA vs methyl]
IF mRNA & protein → mRNA vs protein scatter plot [Protein vs mRNA]
IF treatments → sample order vs treatment IC50 [Tx Waterfall]
